source/cloud/cloud-mobile/troubleshooting-mobile.rst

```diff
@@ -2,49 +2,15 @@
---------------------------------------------------------------------------------------------------------------
-
-Confirm that your server URL has no typos and that it includes ``https://``.
-
 Login with ADFS/Office365 is not working
 ----------------------------------------
 
 In line with Microsoft guidance we recommend `configuring intranet forms-based authentication for devices that do not support WIA <https://docs.microsoft.com/en-us/windows-server/identity/ad-fs/operations/configure-intranet-forms-based-authentication-for-devices-that-do-not-support-wia>`_.
 
-I see a “Connecting…” bar that does not go away
------------------------------------------------
-
-If your app is working properly, you should see a grey “Connecting…” bar that clears or says “Connected” after the app reconnects.
-
-If you are seeing this message all the time, and your internet connection seems fine, ask your server administrator if the server uses NGINX or another webserver as a reverse proxy. If so, they should check that it is configured correctly for `supporting the websocket connection for APIv4 endpoints <https://docs.mattermost.com/install/install-ubuntu-1804.html#configuring-nginx-as-a-proxy-for-mattermost-server>`__.
-
-Build gets stuck at ``bundleReleaseJsAndAssets``
-------------------------------------------------
-
-As a workaround, you can bundle the ``js`` manually first with
-
-.. code-block:: none
-
-  react-native bundle --platform android --dev false --entry-file index.js --bundle-output android/app/src/main/assets/index.android.bundle --assets-dest android/app/src/main/res/
-
-and then ignore the gradle task with
-
-.. code-block:: none
-
-  ./gradlew assembleRelease -x bundleReleaseJsAndAssets
-
-No image previews available in the mobile app
----------------------------------------------
-
-This can happen if the server running Mattermost has its mime types not set up correctly. A server running Linux has this file located in ``/etc/mime.types``. This might vary depending on your specific OS and distribution.
-
-Some distributions also ship without ``mailcap`` which can result in missing or incorrectly configured mime types.
-
-None of these solve my problem!
+I have a different problem!
 -------------------------------
 
-For more troubleshooting help, `open a new topic in our forums <https://forum.mattermost.org/c/trouble-shoot>`__ with steps to reproduce your issue. If you're an Enterprise Edition subscriber, you may open a support ticket in the `Enterprise Edition Support portal <https://mattermost.zendesk.com/hc/en-us/requests/new>`_.
+For more troubleshooting help, `open a new topic in our forums <https://forum.mattermost.org/c/trouble-shoot>`__ with steps to reproduce your issue. If you're a Cloud Enterprise subscriber, you may open a support ticket in the `Mattermost Support portal <https://mattermost.zendesk.com/hc/en-us/requests/new>`_.
```

Comment:
@adamjclarkson For cloud support should we be directing them to: https://customers.mattermost.com/cloud/contact-us ? Not sure if the forum or zendesk makes sense here

Reply:
Yeah agree https://customers.mattermost.com/cloud/contact-us is a nicer page than the zendesk link. Both reach the same team and are acceptable but the Contact Us page is a better experience.

Reply:
And change the language to "If you're a Cloud subscriber, you may reach our Support team 
via our [Contact Us page](https://customers.mattermost.com/cloud/contact-us)

Reply:
Thanks, @adamjclarkson!